docs/components/console/manage-clusters/cluster-backups.md

```diff
@@ -30,10 +30,7 @@ To create a manual backup, take the following steps:
 To create a scheduled backup, take the following steps:
 
 1. Select the **Backups** tab.
-2. Click **Create schedule**.
-
-![cluster-details](./img/cluster-detail-backups.png)
```

Comment:
We reference this screenshot in the section above, so I don't think we need it here 👍